modules/nf-core/prokka/main.nf

```diff
@@ -1,11 +1,11 @@
 process PROKKA {
-    tag "$meta.id"
+    tag "${meta.id}"
     label 'process_low'
 
     conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/prokka:1.14.6--pl5321hdfd78af_4' :
-        'biocontainers/prokka:1.14.6--pl5321hdfd78af_4' }"
+        'oras://community.wave.seqera.io/library/prokka_openjdk:6ee087933c41335a' :
```

Comment:
For singularity images, I've seen in the discussions that is prefered to use the https link instead of the oras one.

Reply:
I'm going to leave this as-is for now, as I'm not sure how to get an https URL - will fix when I have a bit more time to dig!

Reply:
There is a checkbox when creating the container
![image](https://github.com/user-attachments/assets/c8588a99-0bfc-4119-a031-9aabba17e72a)

Reply:
Should be sorted...!